NuGet.Config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+  </packageSources>
+</configuration>
```

Comment:
What's this file for?

Reply:
Huh... I have no idea, I didn't even know this file existed...

Reply:
It's usually in your %appdata% or something. Let's remove it and see if it builds :)

Reply:
Ok I removed the file and it seems to build and run fine... I guess I'll just remove the file then.